tests/app/forms/test_fields.py

```diff
@@ -153,10 +153,21 @@ def test_radio_field(self):
         expected_choices = [(option['label'], option['value']) for option in radio_json['options']]
 
         self.assertTrue(unbound_field.field_class == SelectField)
+        self.assertTrue(unbound_field.kwargs['coerce'], _coerce_str_unless_none)
         self.assertEquals(unbound_field.kwargs['label'], radio_json['label'])
         self.assertEquals(unbound_field.kwargs['description'], radio_json['guidance'])
         self.assertEquals(unbound_field.kwargs['choices'], expected_choices)
 
+    def test__coerce_str_unless_none(self):
+        # pylint: disable=protected-access
+        self.assertEquals(_coerce_str_unless_none(1), '1')
+        self.assertEquals(_coerce_str_unless_none('bob'), 'bob')
+        self.assertEquals(_coerce_str_unless_none(12323245), '12323245')
+        self.assertEquals(_coerce_str_unless_none('9887766'), '9887766')
+        self.assertEquals(_coerce_str_unless_none('None'), 'None')
+        self.assertEquals(_coerce_str_unless_none(None), None)
+        self.assertNotEquals(_coerce_str_unless_none(None), 'None')
```

Comment:
Do we really need both? implicitly if something equals one thing, it won't equal another. We really don't need to test a string isn't equal to None do we?